Eco system (#351)
gwtproject/gwt-site: src/main/markdown/eco/application.md

# Application Frameworks 

> One of the main tasks, when writing a GWT application is separating the code and managing the state. GWT offers with
> [MVP Activities and Places](/doc/latest/DevGuideMvpActivitiesAndPlaces.html) modules to do that. Over the

Reviewer: ```suggestion
One of the main tasks, when writing a GWT application, is separating the code and managing the state. GWT offers
[MVP Activities and Places](/doc/latest/DevGuideMvpActivitiesAndPlaces.html) modules which do that. Over the
```

time other implementations evolved. 

> ## domino-mvp and domino-historie<a id="domino-mvp-historyt"></a>

Reviewer 2: Wrong spelling of domino-history -historie- and the id had an extra 't' at the end.

Domino-mvp is a small, simple and a very lightweight framework for building both simple and complex applications using

> GWT, it introduce the concept of routing using both tokens and events, also allows the developers to write a modular 

Reviewer: ```suggestion
GWT, it introduces the concept of routing using both tokens and events, also allows the developers to write a modular 
```

application and shared components with any other domino application.

For more information about domino-mvp, see: [domino-mvp at GitHub](https://github.com/DominoKit/domino-mvp)

> Domino-historie is a wrapper for the browser history state API.

Reviewer 2: Same spelling issue

> For more information about domino-history, see: [domino-history at GitHub](https://github.com/DominoKit/domino-history)

Reviewer 2: I would prefer to use the website link as it hosts the documentation now instead of the directly sending to github, here

https://dominokit.com/solutions/domino-history/v1

## Nalu<a id="nalu"></a>

Nalu is a tiny framework that helps you to create GWT based applications quite easily. Using the HTML 5 history for 
routing and navigation, Nalu supports the browser's back-, forward-, and reload-button by default and without any
need to implement anything.

> For more information about domino-history, see: [Nalu at GitHub](https://github.com/NaluKit/nalu)

Reviewer 2: This should be *For more information about Nalu* not domino-history.

## Support

> The GWT project team does not support this projects. In case you have questions or would like to open an issue, please

Reviewer: ```suggestion
The GWT project team does not support these projects. In case you have questions or would like to open an issue, please
```

> contact the contributor of the project!

Reviewer: Maybe you meant maintainers? (same for the other pages)

## Missing Something?

> If you are missing something, following the instructions located [here](add-lib.html).

Reviewer 2: Maybe we phrase this as *If you are missing a library or a framework from the list, please follow the instructions here to add it.
